software/base/src/main/java/org/apache/brooklyn/entity/software/base/lifecycle/MachineLifecycleEffectorTasks.java

```diff
@@ -444,9 +493,6 @@ public void run() {
                 }
             }
             entity().addLocations(ImmutableList.of((Location) machine));
-            if (entity().getAttribute(Attributes.JCLOUDS_PROVISIONING_RUNNING) != null) {
```

Comment:
I'll rename the sensors to include "internal" in their names, as well as in the description.

The reason I didn't like this approach was that if `obtain()` failed, we would wait for the max time before stop continued - there was no way for it to terminate early. I'll add a test for that.